src/app/pages/reports-dashboard/components/reports-global-controls/reports-global-controls.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="reports-toolbar">
   <div class="reports-dash-global-controls" fxLayout="row" fxLayoutAlign="center center">
     <mat-slide-toggle
+      color="primary"
```

Comment:
added to have primary colored slide-toggle, if not it's orange :) 
Maybe I forgot to add it in my prev PR of UI improvements